doc/configuration_options.rst

```diff
@@ -223,6 +223,9 @@ The following options are valid in version ``2`` (beside the generic options):
   * ``package_count``: the minimum number of *binary* packages which must be
     available.
   * ``packages``: a list of package names which must be available.
+  * ``package_percent``: The minimum percentage of *binary* packages which must
+    be available.  If both ``package_count`` and ``package_percent`` are
+    specified, ``package_percent`` takes precedence.
```

Comment:
Could you please clarify in this doc that the percentage is calculated against the total *enabled* packages?

Reply:
Yep, updated the docs to point this out in 5fb4a93.